scripts/build/plugins

```diff
@@ -1,21 +1,28 @@
 #!/usr/bin/env bash
 #
-# Build a static binary for the host OS/ARCH
+# Build plugins examples for the host OS/ARCH
 #
 
-set -eu -o pipefail
+set -eu
 
-source ./scripts/build/.variables
+: "${CGO_ENABLED=}"
+: "${GO_LINKMODE=static}"
+: "${GO_BUILDMODE=}"
+: "${GO_BUILDTAGS=}"
+: "${GO_STRIP=}"
+
+. ./scripts/build/.variables
+
+export CGO_ENABLED=0
```

Comment:
Here we're unconditionally overriding the value that was set before; was that intentional?

Reply:
Ah, I see it was there, but inside the loop; perhaps we should just inline it? (like we did for `GO111MODULE`) instead of exporting?

Reply:
Yes inline sound better for this case